x-pack/plugins/siem/public/cases/components/case_view/index.tsx

```diff
@@ -43,8 +42,10 @@ interface Props {
   userCanCrud: boolean;
 }
 
-const MyWrapper = styled(WrapperPage)`
-  padding-bottom: 0;
+const MyWrapper = styled.div`
+  ${({ theme }) => css`
```

Comment:
`css` should be used only in nested interpolations
https://styled-components.com/docs/api#css

in this case I believe we should use `attr`
https://styled-components.com/docs/api#attrs

```
const MyWrapper = styled.div.attr(({ theme }) => ({
  padding-top: theme.eui.paddingSizes.l,
}))``; // make sure to add backticks at the end
``` 

https://styled-components.com/docs/faqs#when-to-use-attrs

Reply:
that throws errs for me @patrykkopycinski , the attribute is `attrs` and I'm not sure its the right one. What about just:
```
const MyWrapper = styled.div`
  padding-top: ${({ theme }) => theme.eui.paddingSizes.l};
`;
```

Reply:
I was wrong, Thank you @stephmilovic  
```
.attrs(({ theme }) => ({ style: {
  paddingTop: theme.eui.paddingSizes.l,
}))
```
should be used for the values that are changing often or if we want to set a default props, for the static values like `theme` is better to write the way you did